pymc3/tests/test_examples.py

```diff
@@ -23,7 +23,10 @@
 
 from .helpers import SeededTest
 
-matplotlib.use('Agg', warn=False)
+try:
```

Comment:
Instead of a try except we can check against the matplotlib version, right?

Reply:
I think.  Was this changed in 3.3.5? I am finding it hard to identify where the change happened.

Reply:
It was changed  in  3.3